agent/lib/commander.js

```diff
@@ -223,7 +223,7 @@ commands:
             }),
           );
 
-          response = await commands.exec(object.lang, object.code);
+          response = await commands.exec(object.lang, object.code, object.timeout || 30);
```

Comment:
The timeouts are usually in `ms`. Is this `ms` or `minutes`?

Reply:
this get passed all the way to python, it is seconds. we can divide by 1000 for consistency in the yaml

Reply:
ok now in client it is ms